Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3.ps1

```diff
@@ -0,0 +1,1572 @@
+            Use the Get-ExORecipient cmdlet to view existing recipient objects in your organization.
+            This cmdlet returns all mail-enabled objects (for example,
+            mailboxes, mail users, mail contacts, and distribution groups).
+
+            #>
+    }
+
+    [PSObject]
+    EXONewTenantAllowBlockList(
+            [string]$entries,
+            [string]$list_type,
+            [string]$list_subtype,
+            [string]$action,
+            [string]$notes,
+            [bool]$no_expiration,
+            [string]$expiration_date
+    )
+    {
+        $cmd_params = @{ }
+        $entries_array = @()
+        if ($entries.Contains(','))
+        {
+            $entries_array = $entries -split ','
+        }
+        if ($entries -and -not $entries_array)
+        {
+            $entries_array += $entries
+            $cmd_params.Entries = $entries_array
+        }
+        else
+        {
+            $cmd_params.Entries = $entries_array
+        }
+        if ($list_type)
+        {
+            $cmd_params.ListType = $list_type
+        }
+        if ($list_subtype)
+        {
+            $cmd_params.ListSubType = $list_subtype
+        }
+        if ($notes)
+        {
+            $cmd_params.Notes = $notes
+        }
+        if ($no_expiration)
+        {
+            $cmd_params.NoExpiration = $null
+        }
+        if ($expiration_date)
+        {
+            $cmd_params.ExpirationDate = $expiration_date
+        }
+        if ($action -eq "Block")
+        {
+            $cmd_params.Block = $null
+        }
+        if ($action -eq "Allow")
+        {
+            $cmd_params.Allow = $null
+        }
+        $this.CreateSession()
+        $results = New-TenantAllowBlockListItems @cmd_params
+        $this.DisconnectSession()
+        return $results
+        <#
+            .DESCRIPTION
+            Use the New-TenantAllowBlockListItems cmdlet to add new entries to the Tenant Allow/Block Lists for your organization.
+            This cmdlet returns all new entries created, details about them, and their status
+
+        #>
+    }
+
+    [PSObject]
+    EXORemoveTenantAllowBlockList(
+            [string]$entries,
+            [string]$ids,
+            [string]$list_type,
+            [string]$list_subtype
+    )
+    {
+        $cmd_params = @{ }
+        $entries_array = @()
+        $ids_array = @()
+        if ($entries.Contains(','))
+        {
+            $entries_array = $entries -split ','
+        }
+        if ($entries -and -not $entries_array)
+        {
+            $entries_array += $entries
+            $cmd_params.Entries = $entries_array
+        }
+        elseif ($entries_array)
+        {
+            $cmd_params.Entries = $entries_array
+        }
+        if ($ids -and $ids.Contains(','))
+        {
+            $ids_array = $ids -split ','
+        }
+        if ($ids -and -not $ids_array)
+        {
+            $ids_array += $ids
+            $cmd_params.Ids = $ids_array
```

Comment:
can remove this line.
and use it once as :
instead of elseif ($ids_array) 
{
            $cmd_params.Ids = $ids_array
}
as 
if ($ids_array) 
{
            $cmd_params.Ids = $ids_array
}